packages/react-dom-bindings/src/client/validateDOMNesting.js

```diff
@@ -155,20 +179,23 @@ if (__DEV__) {
     listItemTagAutoclosing: null,
     dlItemTagAutoclosing: null,
 
-    resourceFormOnly: true,
+    containerTagInScope: null,
   };
 
-  updatedAncestorInfo = function(oldInfo, tag) {
-    const ancestorInfo = {...(oldInfo || emptyAncestorInfo)};
+  updatedAncestorInfoDev = function(
+    oldInfo: ?AncestorInfoDevType,
+    tag: string,
+  ) {
+    const AncestorInfoDev = {...(oldInfo || emptyAncestorInfoDev)};
```

Comment:
Why is this capitalized. Seems like an odd naming convention?

Reply:
I thought so too. I think it's meant to function like a singleton kind of like how Dispatcher is capitalized. It made my type name worse though so I changed it but then changed it back...

It's not a singleton but in memory layout but how it is accessed it's sort of a single global value so it made sense in that (possibly wildly inaccurate) mental framing and I restored it

Reply:
waitaminute....

Reply:
I did a find replace and capitalized it all myself and then convinced myself it was always that way. brb...

Reply:
🤦